project/ren-yi-wang-david/README.rst

```diff
@@ -0,0 +1,106 @@
+==============================================================
+FTST
+==============================================================
+
+Basic Information
+=================
+FDM Thermal Simulator Tool
+
+GitHub repository: https://github.com/ren-yi-wang-david/FTST
+
+Problem to Solve
+================
+power induced thermal distribution,steady state heat transfer,etc.
+
+Prospective Users
+=================
+this tool is developed for CAE,VLSI,CFD simulation user providing verification ,early stage prediction,thermal-aware co optimization.
+
+System Architecture
+===================
+using FDM to discretize the governing equation of heat transfer
+
+using  LU decomposition for solver
```

Comment:
The direct matrix inversion may not deal with stiff matrix.  But it is OK to start your work with the direct way.